src/open_inwoner/components/templates/components/Button/Button.html

```diff
@@ -5,8 +5,9 @@
         class="{{ classes }}"
         href="{{href}}"
         title="{% firstof title text %}"
-        aria-describedby="{{ text }}"
+        {% if text %}aria-describedby="{{ text }}"{% endif %}
         {% if href|startswith:"http" %}target="_blank"{% endif %}
+        {% if hide_text %}aria-hidden="true"{% endif %}
```

Comment:
Aria-hidden doesn't describe the visual state but whether it's available to assistive technologies, I think this should be removed.

I think the aria-label (or title) should always be available and should describe the link. Whether we hide the text is stylistic but in all cases a textual description should be available to assistive technologies.

 > aria-hidden="true" should not be used on elements that can receive focus. Additionally, since this attribute is inherited by an element's children, it should not be added onto the parent or ancestor of a focusable element.

 > The presence of the aria-hidden attribute hides content from assistive technology but doesn't visually hide anything.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden